packages/tailwindcss/src/css-parser.ts

```diff
@@ -337,10 +341,26 @@ export function parse(input: string) {
       closingBracketStack[closingBracketStack.length - 1] !== ')'
     ) {
       let declaration = parseDeclaration(buffer)
-      if (parent) {
-        parent.nodes.push(declaration)
+      if (declaration) {
+        if (parent) {
+          parent.nodes.push(declaration)
+        } else {
+          ast.push(declaration)
+        }
+      } else if (input.charCodeAt(i - 1) === SEMICOLON || input.charCodeAt(i + 1) === SEMICOLON) {
+        let startSemiColonIdx = i
+        while (input.charCodeAt(startSemiColonIdx - 1) === SEMICOLON) {
+          startSemiColonIdx--
+        }
+        let endSemiColonIdx = i
+        while (input.charCodeAt(endSemiColonIdx) === SEMICOLON) {
+          endSemiColonIdx++
+        }
+        throw new Error(`Unexpected: \`${input.slice(startSemiColonIdx, endSemiColonIdx)}\``)
```

Comment:
I'm not sure if this really brings more value then just saying `unexpected semicolon` haha